debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnector.java

```diff
@@ -90,8 +128,7 @@ protected void validateConnection(Map<String, ConfigValue> configValues, Configu
         final SqlServerConnectorConfig sqlServerConfig = new SqlServerConnectorConfig(config);
 
         if (Strings.isNullOrEmpty(sqlServerConfig.getDatabaseName())) {
-            throw new IllegalArgumentException("Either '" + SqlServerConnectorConfig.DATABASE_NAME
-                    + "' or '" + SqlServerConnectorConfig.DATABASE_NAMES
+            throw new IllegalArgumentException("Either '" + DATABASE_NAME + "' or '" + DATABASE_NAMES
```

Comment:
Rather a configuration error should be added (see connection validation failure in this method below).

Reply:
There is actually already validation of both parameters:

1. For `DATABASE_NAME`:
https://github.com/debezium/debezium/blob/2d14b5b09f4af68f6070fbfc4e4a62cd323290f6/debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java#L235
2. And for `DATABASE_NAMES`: https://github.com/debezium/debezium/blob/2d14b5b09f4af68f6070fbfc4e4a62cd323290f6/debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java#L243

So we just need to early return if any of these fields is invalid as it was done prior to https://github.com/debezium/debezium/pull/2604: https://github.com/debezium/debezium/blob/ade15cd8f3c76eb8cbc0c5240716dd92d6c6821b/debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnector.java#L82-L85